src/styles/components/_header.scss

```diff
@@ -189,7 +190,7 @@ input[type="text"]:focus {
 
   .nav-link {
     padding-bottom: 8px;
-    padding-top: 0px;
+    margin-top: 0;
```

Comment:
Does the margin-top need to have a value of 0? Can you just leave it off?

Reply:
Might want to override the parent's properties, perhaps?

Reply:
Yeah it does, the button size at this point is getting smaller so without specifying 0 it throws the alignment off.